clientlib/src/main/java/com/yelp/nrtsearch/server/grpc/LuceneServerStubBuilder.java

```diff
@@ -28,6 +31,16 @@
 
 /** Easy entrypoint for clients to create a Lucene Server Stub. */
 public class LuceneServerStubBuilder implements Closeable {
+  // Create registries with LZ4 compression support
+  public static final CompressorRegistry COMPRESSOR_REGISTRY =
+      CompressorRegistry.getDefaultInstance();
```

Comment:
Do we need to register the LZ4 codec for compressor as well?

Reply:
The compressor registry is directly mutated. The lz4 codec is registered in the static {} block below.